yarn.lock

```diff
@@ -14372,10 +14370,10 @@ terser-webpack-plugin@^1.1.0, terser-webpack-plugin@^1.4.3:
     webpack-sources "^1.4.0"
     worker-farm "^1.7.0"
 
-terser@3.16.1, terser@^4.1.0, terser@^4.1.2:
-  version "4.1.2"
-  resolved "https://registry.yarnpkg.com/terser/-/terser-4.1.2.tgz#b2656c8a506f7ce805a3f300a2ff48db022fa391"
-  integrity sha512-jvNoEQSPXJdssFwqPSgWjsOrb+ELoE+ILpHPKXC83tIxOlh2U75F1KuB2luLD/3a6/7K3Vw5pDn+hvu0C4AzSw==
+terser@3.16.1, terser@^4.1.2, terser@^4.6.2:
+  version "4.6.2"
```

Comment:
Bisecting the upgrade to `terser` to understand what's happening.

I suspect this is necessary to prevent bugs in certain edge cases but the increase is larger than I would expect.

Reply:
Biggest bump from `4.2.1` to `4.3.0`. Still upgrading to be safe. Next time this happens I'll take a look what exactly happened and if we can help terser by specifying options.

Reply:
@dependabot squash and merge

Reply:
In this release I enabled toplevel by default for cjs format. Can it be related?

Reply:
The bump happens when only updating `terser` as well which is why our docs bundle size also increased. As I said most of the increased happens when bumping `terser` from 4.2.1 to 4.3.0.